### ISSUE TITLE
trained models saved in checkpoints

### ISSUE THREAD

**Issue by antgr** [namisan/mt-dnn#137]
I trained with a dataset, and in ```checkpoints``` folder, there are saved models from each epoch.
But what I would expect is the following: use latest model_<n> from epoch n, as a starting point and in epoch 0 of the new training you will have the metrics that you had in epoch n. (you will continue from where you left it).
Instead of this, in epoch 0 I have worse results than ever. What is happening? Can you elaborate?
It seems like it does not use it. Like it is trained from scratch?
edit: Also, I understand that this supposed to be the fine-tuning face, training using task-specific data.

**Comment by namisan** [comment]
you may need to check your learning rate etc...
Regarding the best model, you may customize by yourself.
